sechub-wrapper-owasp-zap/src/main/java/com/mercedesbenz/sechub/zapwrapper/scan/ZapScanner.java

```diff
@@ -368,6 +374,8 @@ UserInformation setupLoginInsideZapContext() throws ClientApiException {
             return null;
         }
 
+        loginCounter.increment();
```

Comment:
Move the increment before the return that we count the successful login attempts only.